iron-doc-behavior.html

```diff
@@ -151,10 +170,16 @@
       },
 
       _compareElements: function(a, b) {
-        if (a.tagname === b.tagname) {
-          return a.classname == null ? -1 : a.classname.localeCompare(b.classname);
+        if (a.name && b.name && a.name !== b.name) {
```

Comment:
Rewrote this. A little hacky, but this way we are comparing directly against an approximation of the display formatting. WDYT?